spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a book suggestion' do
+      let(:user) { create(:user) }
+      let(:book_suggestion) { create(:book_suggestion, user: user) }
+
+      before do
+        post :create, params: { user_id: user.id,
```

Comment:
you can use factory girl here instead of listing all the attributes individually.  Check the method attributes_for.

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#using-factories